src/test/javascript/integration/webhook_max_attempts_spec.js

```diff
@@ -108,6 +108,7 @@ describe(__filename, () => {
         const response = await hubClientGet(webhookResource, headers);
         expect(getProp('statusCode', response)).toEqual(200);
         const body = getProp('body', response) || {};
+      console.info(body);
```

Comment:
sorry for the nitpick, I think we can indent/or remove this line.

Reply:
Whoops! Not a nit-pick at all. I'll get rid of it.